tests/unit/test_x509.cpp

```diff
@@ -568,3 +568,11 @@ TEST_F(X509Test, testGetSerialNumber)
     EXPECT_EQ("-42", negativeSerialCert.getSerialNumberDecimal());
     EXPECT_EQ(std::vector<uint8_t>{0x2a}, negativeSerialCert.getSerialNumberBinary());
 }
+
+TEST_F(X509Test, testCACheck)
+{
+    EXPECT_EQ(false, _root1_expired->isCA())
```

Comment:
I would consider EXPECT_TRUE and EXPECT_FALSE  instead of asserting equality of boolean variables.